util/audio_utils.py

```diff
@@ -0,0 +1,283 @@
+                        folder=output_folder,
+                        prefix=output_folder,
+                        id=counter,
+                        start_ms=sec_to_millis(
+                            vad_span.start + frame_to_sec(split_start, sampling_rate)
+                        ),
+                        end_ms=sec_to_millis(
+                            vad_span.start + frame_to_sec(split_end, sampling_rate)
+                        ),
+                        dept=dept
+                    )
+                    print(
+                        f"{counter} {segment_split_duration:.2f} {sec_to_millis(vad_span.start + frame_to_sec(split_start, sampling_rate)):.2f} {sec_to_millis(vad_span.start + frame_to_sec(split_end, sampling_rate)):.2f} split"
+                    )
+                    counter += 1
+                elif segment_split_duration > upper_limit:
+                    chop_length = segment_split_duration / 2
+                    while chop_length > upper_limit:
+                        chop_length = chop_length / 2
+                    for j in range(int(segment_split_duration / chop_length)):
+                        segment_split_chop = original_audio_segment[
+                            sec_to_millis(
+                                vad_span.start
+                                + frame_to_sec(split_start, sampling_rate)
+                                + chop_length * j
+                            ) : sec_to_millis(
+                                vad_span.start
+                                + frame_to_sec(split_start, sampling_rate)
+                                + chop_length * (j + 1)
+                            )
+                        ]
+                        save_segment(
+                            segment=segment_split_chop,
+                            folder=output_folder,
+                            prefix=output_folder,
+                            id=counter,
+                            start_ms=sec_to_millis(
+                                vad_span.start
+                                + frame_to_sec(split_start, sampling_rate)
+                                + chop_length * j
+                            ),
+                            end_ms=sec_to_millis(
+                                vad_span.start
+                                + frame_to_sec(split_start, sampling_rate)
+                                + chop_length * (j + 1)
+                            ),
+                            dept=dept
+                        )
+                        print(
+                            f"{counter} {chop_length:.2f} {sec_to_millis(vad_span.start + frame_to_sec(split_start, sampling_rate) + chop_length * j ):.2f} {sec_to_millis(vad_span.start + frame_to_sec(split_start, sampling_rate) + chop_length * ( j + 1 )):.2f} chop"
+                        )
+                        counter += 1
+
+
+def split_audio_files(prefix, ext, audio_dir, dept):
```

Comment:
- add type extension for function args.
- add function return

def split_audio_files(prefix:str, ext:str, audio_dir:str, dept:str):